packages/core-utils/src/time.js

```diff
@@ -95,9 +94,9 @@ export function getUserTimezone() {
  * Formats current time for use in OTP query
  * The conversion to the user's timezone is needed for testing purposes.
  */
-export function getCurrentTime() {
+export function getCurrentTime(timezone = getUserTimezone()) {
```

Comment:
Since we're adding this functionality to this method, it seems like it'd make sense to add it to `getCurrentDate` too.